cluster/loadbalance/consistent_hash.go

```diff
@@ -27,7 +27,9 @@ import (
 	"strconv"
 	"strings"
 )
-
+import (
+	gxsort "github.com/dubbogo/gost/sort"
+)
```

Comment:
```suggestion

import (
	gxsort "github.com/dubbogo/gost/sort"
)

```
i think should add a blank line before and after this block